lib/rrd.ml

```diff
@@ -916,9 +916,15 @@ let xml_to_output rrd output =
     output
 
 module Json = struct
-  let string fmt = Printf.ksprintf (fun msg -> `String msg) fmt
+  let fmt fmt x = Printf.ksprintf (fun msg -> `String msg) fmt x
 
-  let float x = string "%.2f" x
+  let string x = fmt "%s" x
+
+  let float x = string (Utils.f_to_s x)
+
+  let int x = fmt "%d" x
```

Comment:
So we encode even integers are strings, not using the JSON number type? As long as the clients can handle this it should be fine.

Reply:
yes, that's correct. In this case the code maintains current behaviour